pkg/agent/agent.go

```diff
@@ -216,14 +216,6 @@ func (k *Klocksmith) watchUpdateStatus(update func(s updateengine.Status)) {
 
 	go k.ue.ReceiveStatuses(ch, stop)
 
-	// synthesize first message
```

Comment:
How about triggering the get here, but just not writing to the channel (since that's ReceiveStatuses's job)? It feels odd to do one call upon starting a goroutine with ReceiveStatuses.

```
_, err := k.ue.GetStatus()
```